client/src/main/java/org/apache/uniffle/client/impl/ShuffleWriteClientImpl.java

```diff
@@ -976,15 +1012,17 @@ public void unregisterShuffle(String appId, int shuffleId) {
                   ShuffleServerClientFactory.getInstance()
                       .getShuffleServerClient(clientType, shuffleServerInfo, rssConf);
               RssUnregisterShuffleResponse response = client.unregisterShuffle(request);
-              if (response.getStatusCode() != StatusCode.SUCCESS) {
-                LOG.warn("Failed to unregister shuffle to " + shuffleServerInfo);
+              if (response.getStatusCode() == StatusCode.SUCCESS) {
+                LOG.info("Successfully unregistered shuffle from {}", shuffleServerInfo);
+              } else {
+                LOG.warn("Failed to unregister shuffle from {}", shuffleServerInfo);
               }
             } catch (Exception e) {
-              LOG.warn("Error happened when unregistering to " + shuffleServerInfo, e);
+              LOG.warn("Error happened when unregistering from " + shuffleServerInfo, e);
```

Comment:
We can refactor this using `{}`. It will be better.

Reply:
Sadly, string interpolation is not available when exception argument is given.

Reply:
It is available if you are using slf4j:

```
LOG.warn("Error happened when unregistering from {}", shuffleServerInfo, e);
```

Reply:
You are right, javadoc did not imply this works but it does, thanks!